angular-workspace/projects/ni/nimble-angular/src/directives/table-column/anchor/tests/nimble-table-column-anchor-router-link-with-href.directive.spec.ts

```diff
@@ -3,11 +3,14 @@ import { ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testin
 import { Router } from '@angular/router';
 import { CommonModule, Location } from '@angular/common';
 import { RouterTestingModule } from '@angular/router/testing';
+// Cannot import from table directive, because tests is not exported in release
```

Comment:
I think at some point we will want to export our page objects in release. This will enable the vision we've been working on in #265 where client tests can interact with a Nimble page object API instead of having to know internal details about timing.

If it's not too complex, can we try that now? We'd need to figure out where they go: maybe in a test utilities folder? Or we change our tsconfig to exclude tests but include files named *.pageobject*? I haven't poked around yet to see what makes the most sense.

Reply:
If you're able to figure out a good pattern we should update the [Folder Structure section of nimble-components CONTRIBUTING](https://github.com/ni/nimble/blob/main/packages/nimble-components/CONTRIBUTING.md#folder-structure) to describe it.

Reply:
Moved comment to this thread:

Te tests namespace is private, don't make stuff there public
The `testing` namespace is where public test utilities are exported.
Move it to `@ni/nimble-components/dist/esm/table/testing/table.pageobject`

Also add the `tests` and `testing` dirs to the restricted imports list in nimble-angular so folks don't add test dependencies to production code: 
https://github.com/ni/nimble/blob/main/angular-workspace/.eslintrc.js#L11

Reply:
If we can get the `systemlink-lib-angular` build working with just the `nimble-components` export then I think we should do that. I'd rather not delay the build fix on a refactor that might require touching all components in `nimble-angular`.

Reply:
I moved the table page object to `testing`. The Angular anchor column test file is now importing from that directory instead.

Reply:
Looks good. Reminder to add a note about where page objects go to CONTRIBUTING as I mentioned above.

Reply:
Done